src/toil/wdl/versions/v1.py

```diff
@@ -33,4 +48,736 @@ def analyze(self):
+        elif is_context(infix, 'NeqContext'):
+            return self._visitInfix2(infix, '!=')
+        elif is_context(infix, 'LteContext'):
+            return self._visitInfix2(infix, '<=')
+        elif is_context(infix, 'GteContext'):
+            return self._visitInfix2(infix, '>=')
+        elif is_context(infix, 'LtContext'):
+            return self._visitInfix2(infix, '<')
+        elif is_context(infix, 'GtContext'):
+            return self._visitInfix2(infix, '>')
+        # continue down our path
+        return self.visitInfix3(infix)
+
+    def _visitInfix2(self, ctx, operation: str):
+        """
+        :param operation: Operation as a string.
+        """
+        lhs = self.visitInfix2(ctx)
+        rhs = self.visitInfix3(ctx)
+        return f'{lhs} {operation} {rhs}'
+
+    def visitInfix3(self, ctx):
+        """
+        Expression infix3 (add/subtract).
+        """
+        infix = ctx.expr_infix3()
+        if is_context(infix, 'AddContext'):
+            return self._visitInfix3(infix, '+')
+        elif is_context(infix, 'SubContext'):
+            return self._visitInfix3(infix, '-')
+        # continue down our path
+        return self.visitInfix4(infix)
+
+    def _visitInfix3(self, ctx, operation: str):
+        """
+        :param operation: Operation as a string.
+        """
+        lhs = self.visitInfix3(ctx)
+        rhs = self.visitInfix4(ctx)
+        return f'{lhs} {operation} {rhs}'
+
+    def visitInfix4(self, ctx):
+        """
+        Expression infix4 (multiply/divide/modulo).
+        """
+        infix = ctx.expr_infix4()
+        if is_context(infix, 'MulContext'):
+            return self._visitInfix4(infix, '*')
+        elif is_context(infix, 'DivideContext'):
+            return self._visitInfix4(infix, '/')
+        elif is_context(infix, 'ModContext'):
+            return self._visitInfix4(infix, '%')
+        # continue down our path
+        return self.visitInfix5(infix)
+
+    def _visitInfix4(self, ctx, operation: str):
+        """
+        :param operation: Operation as a string.
+        """
+        lhs = self.visitInfix4(ctx)
+        rhs = self.visitInfix5(ctx)
+        return f'{lhs} {operation} {rhs}'
+
+    def visitInfix5(self, ctx):
+        """
+        Expression infix5.
+        """
+        return self._visitExpr_core(ctx.expr_infix5().expr_core())
+
+    def _visitExpr_core(self, expr):
+        """
+        Expression core.
+        """
+        if is_context(expr, 'ApplyContext'):
+            return self.visitApply(expr)
+        elif is_context(expr, 'Array_literalContext'):
+            return self.visitArray_literal(expr)
+        elif is_context(expr, 'Pair_literalContext'):
+            return self.visitPair_literal(expr)
+        # elif is_context(expr, 'Map_literalContext'):
+        #     raise NotImplementedError
+        # elif is_context(expr, 'Object_literalContext'):
+        #     raise NotImplementedError
```

Comment:
Did you mean to leave these in here? I might replace them with a `# TODO: implement XXX and YYY`.